rbi/stdlib/ripper.rbi

```diff
@@ -215,3 +215,11 @@ class Ripper::Filter
   # zero or more of the `Ripper::EXPR_*` constants.
   def state; end
 end
+
+class Ripper::SexpBuilder < ::Ripper
+  sig {returns(String)}
+  def error; end
```

Comment:
For future readers: this method seems to have been added in Ruby 3.0; this method is not present in Ruby 2.7.